01week/rockPaperScissors.js

```diff
@@ -8,10 +8,30 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+rockPaperScissors=(hand1, hand2)=>{
```

Comment:
your function isn't defined. Its small but it causes and error in strict mode. Be sure to declare your function and variables.